### DISCUSSION TITLE
No commands, no custom prompts

### DISCUSSION THREAD

**Discussion by zeropwnz** [logancyang/obsidian-copilot#407] [General]
Hello. Thank you for the wonderful plugin. But I have a problem. In your videos where you demonstrate the plugin's functionality, you use various commands such as spell check, translation, summarization, and others. I don't have these commands. Also, I don't have commands that create custom prompts. How can I get them? Thank you.

**Comment by logancyang** [comment]
You need to have your cursor focused on the note to trigger some of the commands, because they could rely on text selection.

**Reply by zeropwnz** [reply]
I highlight text in my note and here are all the commands that I have
![Obsidian_7JTnGB583t](https://github.com/logancyang/obsidian-copilot/assets/160990728/c4c8d2cb-0ce2-4a23-a838-bc4e0f185174)

**Reply by logancyang** [reply]
That's weird, I just tried and couldn't reproduce. I get the same list only if my cursor is not focused on the note. Any messages in the console?

**Reply by zeropwnz** [reply]
Oh my goodness, I just realized which cursor you were referring to. I thought you were talking about the mouse cursor. Everything is working now. Thank you for your help. Sorry for the trouble. xD

**Reply by logancyang** [reply]
Glad it's working!
